libraries/Microsoft.Bot.Expressions/Expression.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Linq;
 using System.Text;
+using Microsoft.Bot.Expressions.Memory;
 
```

Comment:
Did you really need a new namespace?

Reply:
I create a namespace when i found that when when have this, we can make the caller of DialogStateManager use this interface to make the call. Which perhaps, suggests it can be put in another very low level package. Just to keep it isolated for now.